tests/data_examples/bootstrap/das-payload.json

```diff
@@ -0,0 +1,86 @@
+{
+  "settings": {
+    "expiration": {
+      "root": 365,
+      "targets": 365,
+      "snapshot": 1,
+      "timestamp": 1,
+      "bins": 1
+    },
+    "services": {
+      "number_of_delegated_bins": 4,
+      "targets_base_url": "f/",
+      "targets_online_key": true
+    }
+  },
```

Comment:
I don't think we return anywhere the `settings` information when calling `GET /api/v1/sign`.
We don't even show the `settings` in the example.

Reply:
Okay, then let's leave it.